src/main/flight/imu.c

```diff
@@ -342,3 +342,8 @@ int16_t calculateThrottleAngleCorrection(uint8_t throttle_correction_value)
         angle = 900;
     return lrintf(throttle_correction_value * sinf(angle / (900.0f * M_PIf / 2.0f)));
 }
+
+int16_t calculateThrottleAngleCorrectionAxis(uint8_t throttle_hovering_percent, uint8_t axis){
+    uint16_t liftOffTrust = throttle_hovering_percent * 10; //to bring from 0-100 range to 0-1000 range (thrust input is in the 1000-2000 range)
+    return lrintf( liftOffTrust*sinf(anglerad[axis]) );
```

Comment:
thanks for the review.
It is true that at 90% should be infinite, BUT lets skip for a bit things that we cannot fix.

At 45° deg, we find out out that we should increase out "throttle_hovering_percent" by exactly sin(45°) = 0.70 (or 70%) and this is given by vector math; the increase in not linear.
Then over 45° it will increase even more, but i decided to not "add it" but keep sin() (that limit out grow to 2*liftOffTrust) because we find out many quad fly with the trust stick near 50%, so we are probably already almost at max, and we need to leave some power to maneuvering.

any thought?